TAScheduler/views_methods.py

```diff
@@ -123,7 +369,7 @@ def assignTALecture(self, active_lecture):  # new
         pass
 
     def getTACrseAsgmts(self):
-        pass
+        return TAToCourse.objects.filter(ta=self.database).count()
```

Comment:
Method description says to return "list' -we can use a dictionary- of all the course assignments - not the count of assignments.